relays/bin-substrate/src/cli/relay_messages.rs

```diff
@@ -22,7 +22,26 @@ use crate::cli::{
 use crate::messages_lane::MessagesRelayParams;
 use crate::select_full_bridge;
 use structopt::StructOpt;
-use strum::VariantNames;
+use strum::{EnumString, EnumVariantNames, VariantNames};
+
+/// Relayer operating mode.
+#[derive(Debug, EnumString, EnumVariantNames, Clone, Copy, PartialEq)]
+#[strum(serialize_all = "kebab_case")]
+pub enum RelayerMode {
+	/// The relayer doesn't care about rewards.
+	Altruistic,
+	/// The relayer will deliver all messages and confirmations as long as he's not losing any funds.
+	NoLosses,
```

Comment:
```suggestion
	Rational,
```
I'd vote for `Rational` instead of `NoLosses`. Couple of reasons for that:
1. I find the plural and the "no losses" thing a bit weird. AFAIR we still can't guarantee that in case multiple relayers are running it will indeed never lose any funds.
2. `Rational` as "Rational Economic Actor" in game theory dissertations
3. I find it easier to mentally compare `Altruistic` and `Rational` than `Altruistic` and `NoLosses`. Both are adjectives that describe a behavior.